.travis.yml

```diff
@@ -1,53 +1,96 @@
 language: python
 
-python:
-  - "3.5"
-  - "3.6"
+sudo: true
+
+python: "3.6"
+
+addons:
+  apt:
+    packages:
+      - build-essential
+      - cmake
+      - libltdl7-dev
+      - libgsl0-dev
+      - libncurses5-dev
+      - libreadline6-dev
+      - pkg-config
+      - python-all-dev
+      - python-matplotlib
+      - python-numpy
+      - python-scipy
 
 before_install:
+  # in before_install, we install all the NESTML dependencies
+
+  # install antrl4
   - sudo apt install openjdk-8-jre
   - wget http://www.antlr.org/download/antlr-4.7.1-complete.jar
   - echo \#\!/bin/bash > antlr4
   - echo java -cp \"`pwd`/antlr-4.7.1-complete.jar:$CLASSPATH\" org.antlr.v4.Tool \"\$@\" >> antlr4
   - echo >> antlr4
   - chmod +x antlr4
   - export PATH=$PATH:`pwd`
-  - pip install --upgrade pip enum34 pytest
+
+  # install Python module dependencies
+  - pip install --upgrade pip pytest
   - pip install -r requirements.txt
- 
-  # gsl library to support ode-toolbox stiffness testing
-  - wget http://ftp.wrz.de/pub/gnu/gsl/gsl-2.5.tar.gz
-  - tar -xvzf gsl-2.5.tar.gz
-  - cd gsl-2.5
-  - ./configure && make && sudo make install
-  - cd ..
-  - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
-  - pip install pygsl
-
-  #- pip install matplotlib
-  #- cd ~ && git clone https://github.com/nest/nest-simulator nest_source
-  #- mkdir nest_install
-  #- cd nest_source && mkdir build && cd build
-  #- cmake -DCMAKE_INSTALL_PREFIX=~/nest_install .. && make && make install
-  #- cd $TRAVIS_BUILD_DIR
+
+  # install latest odetoolbox from git master (N.B. GSL/PyGSL installation is skipped as NESTML ignores stiffness test result)
+  - pip uninstall --yes odetoolbox
```

Comment:
Unless we tell it otherwise, pip will grab packages from pypi.org. We want to install straight from our github repository instead, using pip's "git+https://..." syntax: https://github.com/nest/nestml/blob/a8c751ed2d0abbf26f1205cc0b105b8540fd1712/.travis.yml#L40